pkgs/misc/drivers/epsonscan2/default.nix

```diff
@@ -55,26 +55,29 @@ stdenv.mkDerivation {
 
   patches = [
     ./build.patch
+    ./gcc14.patch
     (fetchpatch {
-      url = "https://github.com/flathub/net.epson.epsonscan2/raw/master/patches/epsonscan2-crash.patch";
-      hash = "sha256-srMxlFfnZuJ3ed5veFcJIiZuW27F/3xOS0yr4ywn4FI=";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/0002-Fix-crash.patch?h=epsonscan2";
+      hash = "sha256-rNsFnHq//CJcIZl0M6RLRkIY3YhnJZbikO8SeeC2ktg=";
     })
     (fetchpatch {
-      url = "https://raw.githubusercontent.com/flathub/net.epson.epsonscan2/master/patches/epsonscan2-oob-container.patch";
-      hash = "sha256-FhXZT0bIBYwdFow2USRJl8Q7j2eqpq98Hh0lHFQlUQY=";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/0004-Fix-a-crash-on-an-OOB-container-access.patch?h=epsonscan2";
+      hash = "sha256-WmA8pmPSJ1xUdeBbE8Jzi6w9p96aIOm0erF3T4EQ6VA=";
     })
     (fetchpatch {
-      url = "https://raw.githubusercontent.com/flathub/net.epson.epsonscan2/master/patches/epsonscan2-xdg-open.patch";
-      hash = "sha256-4ih3vZjPwWiiAxKfpLIwbbsk1K2oXSuxGbT5PVwfUsc=";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/0003-Use-XDG-open-to-open-the-directory.patch?h=epsonscan2";
+      hash = "sha256-H3lle1SXkkpbBkozYEwiX0z9oTUubTpB+l91utxH03M=";
     })
   ];
 
   postPatch = ''
+    rm CMakeCache.txt
```

Comment:
fyi that is needed as the currents version archive contains that cmake cache file, thus its unhappy about our build system.